src/Polysemy/Internal/Fail.hs

```diff
@@ -0,0 +1,3 @@
+module Polysemy.Internal.Fail where
```

Comment:
Let's put this in `Polysemy.Fail.Type` instead

Reply:
Ok.